boutdata/squashoutput.py

```diff
@@ -89,6 +92,17 @@ def squashoutput(
         If set to True or 0, use the multiprocessing library to read data in parallel
         with the maximum number of available processors. If set to an int, use that many
         processes.
+    time_split_size : int, optional
+        By default no splitting is done. If an integer value is passed, the output is
+        split into files with length in the t-dimension equal to that value. The outputs
+        are labelled by prefacing a counter (starting by default at 0, but see
+        time_split_first_label) to the file name before the .nc suffix.
+    time_split_first_label : int, default 0
+        Value at which to start the counter labelling output files when time_split_size
+        is used.
+    disable_parallel_write : bool, default False
+        Parallel writing may increase memory usage, so it can be disabled even when
+        reading in parallel by setting this argument to True.
     """
     from boutdata.data import BoutOutputs
```

Comment:
```suggestion
    # use local imports to allow fast import for tab-completion
    from boutdata.data import BoutOutputs
```
(To answer https://github.com/boutproject/boutdata/pull/50/files#r659562430)